### DISCUSSION TITLE
此项目使用PaoPaoDNS+PPGW更好还是搭配OpenWrt呢？

### DISCUSSION THREAD

**Discussion by Hi-Jiajun** [joyanhui/ikuai-bypass#40] [General]
不知道怎么测试，目前在用ikuai-bypass搭配paopaodns使用

**Comment by joyanhui** [comment]
个人建议  有条件   iKuai  分流 到   openwrt  openwrt 内运行ikuai-bypass 和 mosdns


ikuai 建议用 域名和isp(ip) 分流模式（需要3个网口 可以是虚拟），网口不够，可以用端口分流。

mosdns 配置ipv4 优先，上游配合一个自建的doh

mosdns的配置文件参考我尚未完工的 https://github.com/joyanhui/mosdns

**Reply by Hi-Jiajun** [reply]
感谢！我自己再学习一下

**Comment by joyanhui** [comment]
话说 如果不是ikuai的某些功能是非用不可的，或者实在是不会用openwrt   建议直接openwrt + shellcrash 就完了

**Reply by refined-fish** [reply]
确实如此，爱快由于闭源，很多简单的功能都无法实现，比如更新这个规则还要自己手搓一个，但是国内的流控与分流免费的只有用这个，openwrt的分流流控确实没办法🤣
